block/submit_loop_test.go

```diff
@@ -103,11 +106,15 @@ func testSubmitLoopInner(
 		timeSinceLast := time.Since(timeLastProgressT).Milliseconds()
 		require.True(t, timeSinceLast < absoluteMax, "too long since last update", "timeSinceLast", timeSinceLast, "max", absoluteMax)
 
+		pendingBlocks.Store(0)                    // no pending blocks to be submitted
```

Comment:
I think strictly, this should not be 0 every time you submit batch, but I guess it makes no difference as only used for logging